pykokkos/lib/ufunc_workunits.py

```diff
@@ -1,6 +1,513 @@
 import pykokkos as pk
 
 
+@pk.workunit
+def greater_impl_5d_int8(tid: int,
+                              view1: pk.View5D[pk.int8],
+                              view2: pk.View5D[pk.int8],
+                              out: pk.View5D[pk.uint8]):
```

Comment:
usual comments about helping out with formatting apply -- feel free to push in here for all the copy-pasting we need to do b/c we don't have a templating lang